src/hypervisor-integration.adoc

```diff
@@ -164,6 +164,36 @@ present when the implementation supports {cheri_default_ext_name}.
 .Virtual supervisor trap default capability register
 include::img/vstdcreg.edn[]
 
+[#vstval,reftext="vstval"]
+==== Virtual Supervisor Trap Value Register (vstval)
```

Comment:
Do we need this section at all? The H spec clearly states that these registers are analogous to the S-mode versions, and nothing is changing between stval vs vstval, so I did not add them initially

Reply:
well - if we have one we should probably have all of them.
In practice it's enough to say that stval is updated and vstval is updated accordingly which would shorten the doc